include/forum.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file include/forum.php
+ * @brief Functions related to forum functionality *
+ */
+
+/**
+ * @brief This class handles functions related to the forum functionality
+ */
+class forum {
+
+	/**
+	 * @brief Function to list all forums a user is connected with
+	 *
+	 * @param int $uid of the profile owner
+	 * @param boolean $showhidden
+	 *	Show frorums which are not hidden
+	 * @param boolean $lastitem
+	 *	Sort by lastitem
+	 * @param boolean $showprivate
+	 *	Show private groups
+	 *
+	 * @returns array
+	 *	'url'	=> forum url
+	 *	'name'	=> forum name
+	 *	'id'	=> number of the key from the array
+	 *	'micro' => contact photo in format micro
+	 */
+	public static function get_forumlist($uid, $showhidden = true, $lastitem, $showprivate = false) {
```

Comment:
Why is it "static"? And: Since the class name contains "Forum", wouldn't it be better to leave out the "forum" out of the function name?